rcmd/structs.go

```diff
@@ -5,6 +5,9 @@ import (
 	"github.com/spf13/afero"
 )
 
+// VERSION is the current pkgr version
+var VERSION = "0.5.0-alpha"
```

Comment:
The version should still be the CLI version. I was more thinking along the lines of

```
type ExecSettings struct {
	WorkDir string `json:"work_dir,omitempty"`
	PkgrVersion string `json:"pkgr_version,omitempty"`
}
```
then in the CLI you could do something along the lines of

```
// takes a dir + version
es := NewExecSettings(wd, VERSION)
```